src/common/motech-file-upload/motech-file-upload.directive.js

```diff
@@ -0,0 +1,34 @@
+(function () {
+    'use strict';
+
+    angular.module('motech-common')
+        .directive('motechFileUpload', ['$compile', '$timeout', '$http', '$templateCache', motechFileUploadDirective]);
+
+    function motechFileUploadDirective($compile, $timeout, $http, $templateCache) {
+        var templateLoader;
+
+        return {
+            restrict: 'E',
+            replace : true,
+            transclude: false,
+            compile: function (tElement, tAttrs, scope) {
+                var url = '../server/resources/partials/motech-file-upload.html',
```

Comment:
@jredlarski Can you change this to '/common/motech-file-upload/motech-file-upload.html'? Thanks for adding this directive.